data/human/human missions.txt

```diff
@@ -3996,3 +3996,286 @@ mission "Fumbling Mugger"
+			`	Again, you put the ship into reverse and turn around. This is not going at all how it was supposed to.`
+				goto rollcall
+			label east2
+			`	Cautiously, you turn to the east and power forward. Subpods fly ahead, exulting in their agility in the face of the dangerous ocean.`
+			`	After holding your breath for what feels like hours, Karengo radios you. "Knew you'd find your way eventually. What's the next direction?"`
+				goto riddlethree
+			label north2
+			action
+				set "Mussie dead"
+			`	Bubbles rush past the thick windows as the group's subpods scout to the north at your direction. At first, all is quiet. Just as you start to exhale a sigh of relief, twin lightning bolts arc across the waterscape ahead and the ship sensors overload, blinded by an apparent release of electricity. The systems recover, showing subpods returning quickly back to the ship.`
+			`	One is missing; Mussie.`
+			`	You turn the ship back the way you came and pull up the sensor recording to see what happened. The flash of light came from viney seaweed, stretching as far up and down as the ship can detect. Evidently, contact with it released a jolt of electricity that lit up both the vine and Mussie's subpod.`
+			label rollcall
+			branch dead1
+				has "Elias and Taddese dead"
+			`	Karengo sounds a roll call over the radio. Mussie never reports in, and his subpod beacon isn't responding. "Ah, well." says Karengo, without much sympathy. "You all stay here and I'll scout ahead."`
+			`	Only minutes later, he's found a clear path east, and you morosly trail the ship behind.`
+				goto riddlethree
+			label dead1
+			`	Karengo sounds a roll call over the radio. Elias and Taddese are still missing, and now Mussie has failed to report in as well. None of their subpod beacons are responding.`
+			`	"Ah, well." says Karengo, without much sympathy. "You all stay here and I'll scout ahead." Only minutes later, he's found a clear path east, and you morosly trail the ship behind.`
+			label riddlethree
+			`	Karengo reads you the next line. "To the north and to the east \ Track my movements like a beast. \ A quick turn right and forward on, \ Come toward me where danger's gone.`
+			branch dead2
+				has "Mussie dead"
+				has "Elias and Taddese dead"
+			branch dead3
+				has "Mussie dead"
+			choice
+				`	(Go north.)`
+					goto north5
+				`	(Go east.)`
+					goto east5
+				`	(Go south.)`
+			`	You smoothly turn the ship to the right, heading south.`
+			`	"Should be close," radios Karengo. "Fire off a ping." It's almost a relief to send out a pulse to map the surroundings after flying dark for so long.`
+			`	The result, however, is a surprise. The display shows a massive object directly below you...as well as as two leviathan-sized *things* back the way you came, each over 200 meters long. Dozens of sets of flippers run along each 80-meter-tall side and hundreds of human-sized teeth line their jaws.`
```

Comment:
As in, each tooth is the size of a human or the teeth are the size of human teeth?